modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -1,67 +1,24 @@
 = Mule Upgrade Tool
 
-*Latest version: 1.1.0*
-
-include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool-1.1.0.adoc[tag=intro]
-
-The Mule upgrade tool enables you to update or upgrade Mule 4 instances located in different folders.
+include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool.adoc[tag=intro]
```

Comment:
This says it's for upgrading 4.1.5 and later, not 4.1.1
https://github.com/mulesoft/docs-release-notes/blob/latest/modules/ROOT/pages/_partials/mule-upgrade-tool/mule-upgrade-tool.adoc

But this could be included in versions of the doc for Mule releases that have older versions of the tool.

Reply:
Modifying it to include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool-1.1.0.adoc[tag=intro]